libr/include/r_arch.h

```diff
@@ -146,13 +136,11 @@ typedef struct r_arch_t {
 } RArch;
 
 typedef struct r_arch_session_t {
-#if R2_590
 	char *name; // used by .use to chk if it was set already
 	// TODO: name it "peer" instead of encoder. so the encoder can back reference the decoder
```

Comment:
```suggestion
```

Reply:
if i remove this comment then i need to change the variable below and more stuff. better keep that todo for another pr